### ISSUE TITLE
Shell expansion syntax not recognized for imports

### ISSUE THREAD

**Issue by fabiomoratti** [linux-china/jetbrains-just-plugin#31]
Minor issue but could be worth fixing.

The following code is marked as a syntax error by IntelliJ (PyCharm 2024.1.4)

```
import x'envs/${JUST_COMMON:-common}.just'
```

With the following error:  `JustToken.IMPORT_PATH or JustToken.QUESTION_MARK expected, got 'x'` but the expansion works as expected as documented at the bottom of the page [here](https://just.systems/man/en/chapter_30.html): 

> _However, this allows shell expanded strings to be used in places like settings **and import paths**, which cannot depend on just variables and .env files._

Note that the shell expansion is correctly recognized for settings, the code: 

```
set dotenv-filename := x'envs/${JUST_ENV:-dummy}.env'
```

does not trigger a syntax error.

**Comment by linux-china** [comment]
It's fixed on 0.5.2, and will be released ASAP. 
![Snip20240926_12](https://github.com/user-attachments/assets/5ca89c95-4a21-4397-971d-1212b2a086a4)

**Comment by linux-china** [comment]
Fixed with version 0.5.2
